.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: node_js
 node_js:
-  - "iojs"
-  - "0.12"
-  - "4"
-  - "5"
   - "6"
+  - "7"
+  - "8"
+  - "9"
+  - "10"
 addons:
-  - "firefox": "36.0.4"
+  - "firefox": latest-esr
 install:
   - "npm install"
   - "./node_modules/bower/bin/bower install"
 before_script:
   - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
```

Comment:
not needed anymore, because we're using [firefox headless mode](https://developer.mozilla.org/fr/docs/Mozilla/Firefox/Headless_mode)